index.js

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2017 OCAD University
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/fluid-couch-config/master/LICENSE.txt
+*/
+
+"use strict";
+
+/* eslint-env node */
+var fluid = require("infusion");
+
+require("./src/couchConfig");
+
+// Register our content so that it can be referenced in other packages using `fluid.module.resolvePath("%gpii-binder/path/to/content")`
+fluid.module.register("couch-config", __dirname, require);
```

Comment:
Let's update the module name to `"fluid-couch-config"`.

And it looks like the comment on line 17 was copied from gpii-binder.

Reply:
For the comment, would it be appropriate simply to replace the reference to the gpii-binder module with this module's name?
i.e. `fluid.module.resolvePath("%fluid-couch-config/path/to/content")`

Reply:
I would say either update as you suggest, or we could also just remove it.